app/routes/editor/new.js

```diff
@@ -36,6 +37,12 @@ export default AuthenticatedRoute.extend(base, {
         psm.send('resetPubDate');
 
         this._super(...arguments);
+
+        controller.set('cards' , []);
+        controller.set('atoms' , []);
+        controller.set('toolbar' , []);
+        controller.set('apiRoot', ghostPaths().apiRoot);
```

Comment:
it would probably be better to inject the `ghostPaths` service into the controller rather than re-set the API and asset paths each time the route is entered.